app/component/DeparturesRow.js

```diff
@@ -0,0 +1,253 @@
+            {
+              id: 'departure-page-sr',
+            },
+            {
+              shortName,
+              destination: headsign,
+              time: localizeTime(dep.stoptime * 1000),
+            },
+          )}
+        />
+        {node}
+      </>
+    );
+  };
+
+  const departureTimeWithLink = dep => {
+    const departureTime = dep.stoptime;
+    const shownTime = getShownTime(departureTime);
+    return renderWithLink(
+      <>
+        <span
+          className={cx('route-time', {
+            realtime: dep.realtime,
+            canceled,
+          })}
+          aria-hidden="true"
+        >
+          {dep.realtime ? '' : '*'}
+          {shownTime}
+        </span>
+        <span className="sr-only">
+          {intl.formatMessage(
+            {
+              id: 'departure-time-sr',
+            },
+            {
+              when: shownTime,
+              time: localizeTime(departureTime * 1000),
+              realTime: dep.realtime
+                ? intl.formatMessage({ id: 'realtime' })
+                : '',
+            },
+          )}
+        </span>
+      </>,
+      dep,
+    );
+  };
+
+  const departureTimes = deps => {
+    return deps.map((dep, index) => (
+      <>
+        {index > 0 ? ', ' : ''}
+        {departureTimeWithLink(dep)}
+      </>
+    ));
+  };
+
+  let icon;
+  let iconColor;
+  let backgroundShape;
+  let sr;
+
+  if (route?.alerts?.length > 0) {
+    const alert = getMostSevereAlert(route, trip);
+    sr = (
+      <span className="sr-only">
+        {intl.formatMessage({
+          id: 'disruptions-tab.sr-disruptions',
+        })}
+      </span>
+    );
+    icon =
+      alert.alertSeverityLevel !== 'INFO'
+        ? 'icon-icon_caution-white-excl-stroke'
+        : 'icon-icon_info';
+    iconColor = alert.alertSeverityLevel !== 'INFO' ? '#DC0451' : '#888';
+    backgroundShape = 'circle'; // alert.alertSeverityLevel !== 'INFO' ? undefined : 'circle';
+  }
+
+  return (
+    <tr
+      className={cx(
+        'departure-row',
+        mode,
+        departure.bottomRow ? 'bottom' : '',
+        props.className,
+      )}
+      key={uuid()}
+    >
+      <td
+        className={cx('route-number-container', {
+          long: shortName.length <= 6 && shortName.length >= 5,
+        })}
+        style={{ border: `1px solid #${departure.trip.route.color}` }}
+      >
+        {renderWithLink(
+          <>
+            <div className="route-number">{shortName}</div>
+            {icon && (
+              <>
+                <Icon
+                  className={backgroundShape}
+                  img={icon}
+                  color={iconColor}
+                  backgroundShape={backgroundShape}
+                />
+                {sr}
+              </>
+            )}
+          </>,
+          departures[0],
+          true,
+        )}
+      </td>
+      <td className={cx('route-headsign', departure.bottomRow ? 'bottom' : '')}>
```

Comment:
This property (copied from upstream's `DepartureRow`) should perhaps be named differently, as it contains elements to show in case only drop off is possible. I'll not change here, to stay in line with upstream